components/Auth/auth.css

```diff
@@ -0,0 +1,20 @@
+.auth-container{
+    margin: 4rem 1rem 2rem 1rem;
```

Comment:
try to keep the margins and padding consistent and avoid being very specific.

Reply:
Done. 👍